config/environments/production.rb

```diff
@@ -134,7 +134,7 @@
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = (ENV['ENABLE_HTTPS'] == 'yes')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.